determined-ai/determined
# fix: replace `InlineEditor` with UIKit input

## Description
[WEB-850]




## Test Plan
- Go to setting page, and check if a user can edit username and display name
  - design is slightly changed. User can now confirm/cancel the current change
- Go to experiment detail page, click the menu dropdown, and check if  a user can edit experiment name
  - If a user doesn't have permissions to edit experiment, `edit` button is not visible
  - Check if user cannot edit name, description, and tag by clicking experiment name, description, and tag
- Go to model detail page, click the menu dropdown, and check if  a user can edit model name
  - If a user doesn't have permissions to edit model, `edit` button is disabled
  - Check if user cannot edit name by clicking model name
- Go to model version detail page, click the menu dropdown, and check if  a user can edit model name
  - If a user doesn't have permissions to edit model version, `edit` button is disabled
  - Check if user cannot edit name by clicking model version name




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket






[WEB-850]: https://determinedai.atlassian.net/browse/WEB-850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
